packages/editor/src/components/provider/index.native.js

```diff
@@ -150,6 +151,12 @@ class NativeEditorProvider extends Component {
 				this.props.createSuccessNotice( payload.message );
 			}
 		);
+
+		this.subscriptionParentShowEditorHelp = subscribeShowEditorHelp( () => {
+			this.props.createSuccessNotice(
+				'Show Editor Help request received by JS!'
+			);
```

Comment:
This is temporary and only implemented to give some sort of feedback while testing these changes. The actual handling of the request to open the help topics bottom sheet will be implemented in the next PR.

Reply:
WDYT about placing an inline comment stating this is temporary code? Do you think that is worthwhile?

Reply:
Comment placed 👍